Fix:
error: implicit declaration of function 'atoi' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2009-03-03 17:38:38.000000000 -0600
+++ configure	2024-01-30 17:59:18.000000000 -0600
@@ -27675,6 +27675,7 @@
 cat >>conftest.$ac_ext <<_ACEOF
 /* end confdefs.h.  */
 
+#include <stdlib.h>
 #include <string.h>
 #include <zlib.h>
 
